test/misc/addresssettings/setup.go

```diff
@@ -47,70 +46,21 @@ var _ = ginkgo.AfterEach(func() {
 	sw.AfterEach()
 })
 
-func verifyAddressSettingsInt(address, AddressBit, itemName string,
-	value int, hw *test_helpers.HttpWrapper) {
+func retrieveAddressSettings(address, AddressBit string, hw *test_helpers.HttpWrapper) test_helpers.Value {
 	domain := strings.Split(address, ".")[0]
 	header := strings.Replace(OriginHeader, "NAME", domain, 1)
 	hw.AddHeader("Origin", header)
-	actualURL := "http://admin:admin@" + address + CallAddress + AddressBit
+	actualURL := "http://" + address + CallAddress + AddressBit
+    log.Logf("url (string): %s", actualURL)
 	hw.WithPassword("admin").WithUser("admin")
```

Comment:
hardcoded user and password. will be better if it is configurable using parameters or config file or at least defined as a constant in the code.